src/ui/processPicker.ts

```diff
@@ -0,0 +1,252 @@
+/*---------------------------------------------------------
```

Comment:
This is lightly modified code from the existing debug adapter, primarily just removing support for the legacy `--debug` protocol and protocol detection.